src/App.scss

```diff
@@ -65,3 +65,8 @@
     padding-right: 0px;
   }
 }
+
+.text-small {
```

Comment:
These kind of utility classes aren't as good. They describe what something looks like rather than what it is. This is sometimes okay, but it usually makes it harder to theme because it gives you less control when you are doing theme overrides. It's better to make a scss variable or mixin that you can reuse for multiple classes.